src/save_index.hpp

```diff
@@ -124,6 +124,8 @@ class save_index_class : public std::enable_shared_from_this<save_index_class>
 	config& data();
 
 	static void fix_leader_image_path(config& data);
+	/** Deletes non-existent save files from the index. **/
```

Comment:
Although Doxygen comments need `/**` at the start, they only need `*/` at the end.